src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketsTable.tsx

```diff
@@ -0,0 +1,246 @@
+import { useEffect, useRef } from "react";
+import { useMutation } from "@apollo/client";
+import styled from "@emotion/styled";
+import Button from "@leafygreen-ui/button";
+import Icon, { Size } from "@leafygreen-ui/icon";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Table } from "antd";
+import { useAnnotationAnalytics } from "analytics";
+import { ConditionalWrapper } from "components/ConditionalWrapper";
+import { Popconfirm } from "components/Popconfirm";
+import { useToastContext } from "context/toast";
+import {
+  MoveAnnotationIssueMutation,
+  MoveAnnotationIssueMutationVariables,
+  RemoveAnnotationIssueMutation,
+  RemoveAnnotationIssueMutationVariables,
+} from "gql/generated/types";
+import { MOVE_ANNOTATION, REMOVE_ANNOTATION } from "gql/mutations";
+import { AnnotationTicketRow } from "./AnnotationTicketRow";
+import { AnnotationTickets, AnnotationTicket } from "./types";
+
+interface AnnotationTicketsProps {
+  jiraIssues: AnnotationTickets;
+  taskId: string;
+  execution: number;
+  isIssue: boolean;
+  userCanModify: boolean;
+  selectedRowKey: string;
+  setSelectedRowKey: (key: string) => void;
+  loading: boolean;
+}
+
+const AnnotationTicketsTable: React.VFC<AnnotationTicketsProps> = ({
+  jiraIssues,
+  taskId,
+  execution,
+  userCanModify,
+  isIssue,
+  selectedRowKey,
+  setSelectedRowKey,
+  loading,
+}) => {
+  const annotationAnalytics = useAnnotationAnalytics();
+  const dispatchToast = useToastContext();
+  const issueString = isIssue ? "issue" : "suspected issue";
+
+  // Once JIRA tickets have been fetched, display the complete information.
```

Comment:
nit. You can remove this comment since it only makes sense in the previous version of this file